src/vs/workbench/api/common/extHostSCM.ts

```diff
@@ -757,7 +760,7 @@ export class ExtHostSCM implements ExtHostSCMShape {
 		});
 
 		const handle = ExtHostSCM._handlePool++;
-		const sourceControl = new ExtHostSourceControl(extension, this._proxy, this._commands, id, label, rootUri);
+		const sourceControl = new ExtHostSourceControl(extension, this._proxy, this._commands, id, label, rootUri ? normalizePath(rootUri.with({ path: toPosixPath(normalizeDriveLetter(rootUri.fsPath)) })) : undefined);
```

Comment:
@joaomoreno I have no idea if this is safe to do for all of SCM. I can move it into the quick diff service if that would be better.